distsql/select_result.go

```diff
@@ -96,6 +96,7 @@ func (r *selectResult) fetch(ctx context.Context) {
 
 		close(r.results)
 		duration := time.Since(startTime)
+		// TODO: Add a label to distinguish between success or failure
```

Comment:
how about filing a github issue for this TODO action?

Reply:
https://github.com/pingcap/tidb/issues/11397